server/src/domain/model/mod.rs

```diff
@@ -1,5 +1,6 @@
 pub mod prelude;
 
+pub mod deserialize;
```

Comment:
Should it be `pub(crate)` ?

So far I haven't used it much because I mostly had one crate, but now it might make sense to start tightening the visibility, especially around everything DB related.

Reply:
Yeah, it might as well be. I'll just do it for deserialize now, I'd prefer to not mix a general tightening in right now.